frontend/packages/cli/src/cli/erdCommand/buildCommand/index.ts

```diff
@@ -56,5 +56,14 @@ export const buildCommand = async (
     )
   })
 
+  if (errors.length === 0) {
+    console.info(`
+ERD has been generated successfully in the \`dist/\` directory.
+Note: You cannot open this file directly using \`file://\`.
+Please serve the \`dist/\` directory with an HTTP server and access it via \`http://\`.
+Example:
+  npx http-server dist/
```

Comment:
nits: It would be good to change the color just here. What do you think?

Reply:
👀 Did you mean something like this? @MH4GF 


```suggestion
    ${blueBright("$ npx http-server dist/")}
```

image:

<img width="766" alt="スクリーンショット 2025-01-17 17 29 11" src="https://github.com/user-attachments/assets/059df5ec-9fa1-4a14-8f42-b7f0cf2fd7cb" />

Reply:
Looks good! I think the user can visually understand the next action.

Reply:
https://github.com/liam-hq/liam/pull/496/commits/39441880271e9563fcdf9d1786490dbcd6934374 thanks,  modified!